akka-projection-slick/src/main/scala/akka/projection/slick/SlickProjection.scala

```diff
@@ -49,16 +58,58 @@ object SlickProjection {
       projectionId: ProjectionId,
       sourceProvider: SourceProvider[Offset, Envelope],
       databaseConfig: DatabaseConfig[P],
-      handler: SlickHandler[Envelope]): ExactlyOnceSlickProjection[Envelope] =
+      handler: SlickHandler[Envelope])(
+      implicit system: ActorSystem[_]): ExactlyOnceSlickProjection[Offset, Envelope] = {
```

Comment:
In the beginning I was trying to avoid requiring an ActorSystem too early. When building the projection it was not necessary and we could defer it to when it get run, but to build a lifted handler we do need it. 

This is not a big deal, the user will have an actor system around, so maybe we could standardise and request an ActorSystem when building it for all projections. Then we can remove it from the methods (ie: `mappedSource` and `run`).